src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/AnswerRatingResource.java

```diff
@@ -147,20 +146,18 @@ public Rating putRating(Request request, Response response) {
             throw new IllegalArgumentException("Quality cannot be set when creating a Rating");
         }
 
-        RatingRecord r = null;
+        Rating result;
+
         try {
-            r = answerRatingOperations.insertNewRating(
-                    AnswerRatingTransformer.toRatingRecord(rating, answerId, experimentId)
-            );
+            result = answerRatingOperations.insertNewRating(rating, answerId, experimentId);
         } catch (IllegalArgumentException | IllegalStateException e) {
             throw new BadRequestException(e.getMessage());
         }
 
         response.status(201);
+        response.header("Location", "/experiments/" + experimentId + "/answers/" + answerId);
```

Comment:
öhm this creates a new rating and you are pointing the new location to a answer, isnt this wrong ?